third-party/genuine-fantasy-press/forgotten-secrets/archetypes/warlock-storm-lord.xml

```diff
@@ -0,0 +1,146 @@
+            <p class="indent">Additionally, whenever you take the Dash action, nonmagical ranged attacks that target you are made with disadvantage until the start of your next turn.</p>
+		</description>
+		<sheet>
+			<description>
+				You passively manifest a small tempest around yourself. You gain advantage on saving throws against dangerous gases, clouds, forceful winds, and other similar effects. Your vision is not obscured by fog, mist, smoke, rain, clouds, or other airborne particles. Additionally, whenever you take the Dash action, nonmagical ranged attacks that target you are made with disadvantage until the start of your next turn.
+			</description>
+		</sheet>
+	</element>
+
+	<element name="Rider of Lightning" type="Archetype Feature" source="Player’s Handbook" id="ID_GFP_COFSA_ARCHETYPE_FEATURE_STORMLORD_RIDEROFLIGHTNING">
+		<description>
+			<p>Starting at 10th level, when you deal lightning or thunder damage using a spell of 1st level or higher or cast a spell that is on your patron’s extended spell list, you can use your bonus action to teleport to an unoccupied space adjacent to a target of the spell or to an unoccupied space within the spell’s area of effect.</p>
+		</description>
+		<sheet>
+			When you deal lightning or thunder damage using a spell of 1st level or higher or cast a spell that is on your patron’s extended spell list, you can use your bonus action to teleport to an unoccupied space adjacent to a target of the spell or to an unoccupied space within the spell’s area of effect.
+		</sheet>
+	</element>
+
+	<element name="Slayer of the King" type="Archetype Feature" source="Player’s Handbook" id="ID_GFP_COFSA_ARCHETYPE_FEATURE_STORMLORD_SLAYEROFTHEKING">
+		<description>
+			<p>At 14th level, you discover the unspoken secrets of the heroes who sought the downfall of the Storm Lord. As an action, you cleave the air with a blade of purest wind in a line 300 feet long, 300 feet high, and 5 feet wide. Each creature in the line must make a Dexterity saving throw against your warlock spell save DC. A creature takes force damage equal to three times your warlock level on a failed save or half as much damage on a successful one. Unattended objects automatically fail this saving throw.<p>
+            <p class="indent">Once you use this feature, you can’t do so again until you finish a long rest.</p>
+		</description>
+		<sheet>
+			As an action, you cleave the air with a blade of purest wind in a line 300 feet long, 300 feet high, and 5 feet wide. Each creature in the line must make a Dexterity saving throw against your warlock spell save DC. A creature takes {{slayer of the king:dmg}} force damage or half as much damage on a successful one. Unattended objects automatically fail this saving throw. Once you use this feature, you can’t do so again until you finish a long rest.
+		</sheet>
+        <rules>
+            <stat name="slayer of the king:dmg" value="level:warlock" >
```

Comment:
Supposedly this line -> line 142 are missing a closing /, as it outputs an error and in other patrons the "stat" line has a /, but when I added them Aurora refused to load and doesn't spit out an error. No idea what happened there.